tutorials/3d/standard_material_3d.rst

```diff
@@ -1,18 +1,19 @@
 .. _doc_standard_material_3d:
 
-Standard Material 3D
-====================
+Standard Material 3D and ORM Material 3D
+========================================
 
 Introduction
 ------------
 
-``StandardMaterial3D`` is a default 3D material that aims to provide most of the features
-artists look for in a material, without the need for writing shader code. However,
-it can be converted to shader code if additional functionality is needed.
+``StandardMaterial3D`` and  ``ORMMaterial3D`` (Occlusion, Roughness, Metallic)
+are default 3D materials that aim to provide most of the features artists look
+for in a material, without the need for writing shader code. However, they can
+be converted to shader code if additional functionality is needed.
 
-This tutorial explains most parameters present in ``StandardMaterial3D``.
+This tutorial explains the parameters present in both materials.
 
-There are three ways to add a ``StandardMaterial3D`` to an object. It can be added in
+There are three ways to add a these materials to an object. It can be added in
```

Comment:
```suggestion
There are three ways to add a these materials to an object. A material can be added in
```

Making this change as the previous sentence now uses plural ("these materials"), while the following sentences still use singular ("it can be added"). Hope this change makes it a bit clearer :) We could also simply change "it" to plural "they" but that seems a bit confusing, maybe.